manager.go

```
@@ -13,46 +13,46 @@ const (
 	err error // latest error
 
 	// name of the manager
 	name string
 	// pool sync.Pool
 	// is a sample for new BasicEvent
-	sample *BasicEvent
+	sample *BasicEvent[T]
 
 	// storage user custom Event instance. you can pre-define some Event instances.
-	events map[string]Event
+	events map[string]Event[T]
```

golangci-lint (1.64.8)

field `events` is unused

(unused)